lib/voicebase/v2/client.rb

```diff
@@ -20,7 +20,7 @@ def authenticate!
               'Accept'       => 'application/json'
             }
           ), api_version)
-        @token = VoiceBase::Client::Token.new(response.tokens.try(:first).try(:[], 'token'))
+        @token = VoiceBase::Client::Token.new(response.tokens.any? && response.tokens.first.fetch("token"))
```

Comment:
The reason the part `response.tokens.try(:first)` is there because `response.tokens` could be nil. If I do `nil.any?` in IRB I get a NoMethodError. It might be a good idea to add a spec for this case.